matrix/max_area_of_island.py

```diff
@@ -67,60 +67,58 @@ def count_max_area(mat: list[list[int]]) -> int:
     return max_area
 
 
-# Example 1:
-mat = [
-    [0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
-    [0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0],
-    [0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0],
-    [0, 1, 0, 0, 1, 1, 0, 0, 1, 0, 1, 0, 0],
-    [0, 1, 0, 0, 1, 1, 0, 0, 1, 1, 1, 0, 0],
-    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0],
-    [0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0],
-    [0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0],
-]
-
-
-print(count_max_area(mat))  # Output -> 6
-
-"""
-Explanation:
-We are allowed to move 4-directionally (horizontal or vertical.)
-so the possible
-in a matrix if we are at x and y position the possible moveing are
-
->> directions = [(x,y+1),(x,y-1),(x+1,y),(x-1,y)]
-also we need to care of boundary cases as well
-which are x and y can not be smaller than 0 and greater than number
-of rows and columns respectively.
-
-
-Visualization
-mat = [
-  [0,0,A,0,0,0,0,B,0,0,0,0,0],
-  [0,0,0,0,0,0,0,B,B,B,0,0,0],
-  [0,C,C,0,D,0,0,0,0,0,0,0,0],
-  [0,C,0,0,D,D,0,0,E,0,E,0,0],
-  [0,C,0,0,D,D,0,0,E,E,E,0,0],
-  [0,0,0,0,0,0,0,0,0,0,E,0,0],
-  [0,0,0,0,0,0,0,F,F,F,0,0,0],
-  [0,0,0,0,0,0,0,F,F,0,0,0,0]
-]
-
-For visualization I have defined the connected island with alphabates.
-by observation we can see that
-    A island is of area 1
-    B island is of area 4
-    C island is of area 4
-    D island is of area 5
-    E island is of area 6 and
-    F island is of area 5
-
-it has 6 unique island's of mentioned area's
-and maximum of all of them is 6 so we return 6.
-"""
-
-
 if __name__ == "__main__":
     import doctest
 
     doctest.testmod()
+
+    # Example 1:
+    mat = [
+        [0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0],
+        [0, 1, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 1, 0, 0, 1, 1, 0, 0, 1, 0, 1, 0, 0],
+        [0, 1, 0, 0, 1, 1, 0, 0, 1, 1, 1, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0],
+    ]
+
+    print(count_max_area(mat))  # Output -> 6
+
+    """
+    Explanation:
+    We are allowed to move 4-directionally (horizontal or vertical.)
+    so the possible
+    in a matrix if we are at x and y position the possible moveing are
+
+    >> directions = [(x,y+1),(x,y-1),(x+1,y),(x-1,y)]
+    also we need to care of boundary cases as well
+    which are x and y can not be smaller than 0 and greater than number
+    of rows and columns respectively.
+
+
+    Visualization
+    mat = [
+    [0,0,A,0,0,0,0,B,0,0,0,0,0],
+    [0,0,0,0,0,0,0,B,B,B,0,0,0],
+    [0,C,C,0,D,0,0,0,0,0,0,0,0],
+    [0,C,0,0,D,D,0,0,E,0,E,0,0],
+    [0,C,0,0,D,D,0,0,E,E,E,0,0],
+    [0,0,0,0,0,0,0,0,0,0,E,0,0],
+    [0,0,0,0,0,0,0,F,F,F,0,0,0],
+    [0,0,0,0,0,0,0,F,F,0,0,0,0]
```

Comment:
Could you make the formatting in your block here (and maybe the entire docstring you have here) match our code style?